index.js

```diff
@@ -28,4 +28,24 @@ module.exports = (api, opts) => {
         }
       })
   })
+
+  //Add bootstrap's variables globally
+      const bootstrapVueVarImports = [
```

Comment:
Are we sure this is needed? What use cases does this fulfill?

Reply:
Yes, it allows using all bootstrap's variables, mixins and functions in vue components files.
For example, I use a lot the media query mixins (media-breakpoint-up, media-breakpoint-down etc), if I don't inject the variables and mixins in all vue components, sass-loader will not be able to resolve those mixins.
Other examples can be, using color(), theme-color(), theme-color-level(), gray() helper functions.